openshift/template.yaml

```diff
@@ -66,27 +66,6 @@ parameters:
 - name: AGENT_DOCKER_IMAGE
   value: ''
   required: true
-- name: CONNECTIVITY_CHECK_IMAGE
```

Comment:
this will break the integration environment until we will change the saas.yaml so I think maybe it will be better to change it to required: false and after we will delete it from the saas.yaml - let's remove it from the code